src/Uno.UI/UI/Xaml/DependencyObjectStore.cs

```diff
@@ -792,25 +790,24 @@ private void WritePropertyEventTrace(int eventId, DependencyProperty property, D
 		[MethodImpl(MethodImplOptions.AggressiveInlining)]
 		private void ValidatePropertyOwner(DependencyProperty property)
 		{
-			if (_validatePropertyOwner)
-			{
-				var isFrameworkElement = _originalObjectType.Is(typeof(FrameworkElement));
-				var isMixinFrameworkElement = _originalObjectRef.Target is IFrameworkElement && !isFrameworkElement;
+#if DEBUG
```

Comment:
The original idea here is that if the debugger is attached, even in release builds that come from nuget, this check is still done for app developers. Changing it for a compile time conditional will remove that check.

Reply:
Ah, I see. Good point. Can we make this a FeatureConfigutarion option instead? We can explicitly keep the checks internally for SamplesApp (always) and make the public builds run the checks if Debugger.IsAttached. Actually, I would prefer to make the checks run if `Debugger.IsAttached` OR `DEBUG`, as these checks are really useful when developping even when a debugger isn't attached.

Reply:
Sure, a feature configuration would be fine indeed.